# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/compiler/translator/ExtensionBehavior.cpp
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/compiler/translator/ExtensionBehavior.cpp
--- a/gfx/angle/checkout/src/compiler/translator/ExtensionBehavior.cpp	Sat Sep 07 13:36:46 2019 +0000
+++ b/gfx/angle/checkout/src/compiler/translator/ExtensionBehavior.cpp	Sat Sep 07 13:38:36 2019 +0000
@@ -29,8 +29,11 @@
     OP(OES_EGL_image_external_essl3)             \
     OP(OES_standard_derivatives)                 \
     OP(OES_texture_storage_multisample_2d_array) \
+    OP(OES_texture_3D)                           \
+    OP(OVR_multiview)                            \
     OP(OVR_multiview2)                           \
-    OP(ANGLE_multi_draw)
+    OP(ANGLE_multi_draw)                         \
+    OP(ANGLE_base_vertex_base_instance)
 
 namespace sh
 {